### PR TITLE
feat: added some more variants in alert and avatar components

### PR DESCRIPTION
**Pull request by Manju-sree**
Added some more variants in alert and avatar components, please review

Alert
- Alerts with icon and close button
- Example of Alert bar with close button 
- Alerts with outline and icon

Avatar
- Example of Square Avatar
- Example of Text Avatar

### CODE REVIEW

File: Components/Alert/alert.html [hunk 1]
```diff
@@ -48,43 +50,91 @@ <h2 class="side-title ">Components</h2>
...
+                        <strong>secondary</strong> alert - check it out!
                     </div>
                     <div class="success alert-box ">
-                        <span><i class="fas fa-check-circle "></i></span> success alert - check it out!
+                        success alert - check it out!
                     </div>
                     <div class="Warning alert-box ">
-                        <span><i class="fas fa-exclamation-triangle "></i></span> waring alert - check it out!
+                        waring alert - check it out!
+                    </div>
```
Comment (mariyasada) [inline Components/Alert/alert.html]: fix this typo to warning alert

**Review by mariyasada — APPROVED** [review]
fix that typo. you can merge

File: Components/Alert/alert.html (hunk 1 above)
Comment (Manju-sree) [reply Components/Alert/alert.html]: thanks for letting me know this

File: Components/Alert/alert.css [hunk 2]
```diff
@@ -1,35 +1,70 @@
...
+    margin: 1rem 0;
+    position: relative;
+    width: 50rem;
+    font-size: 1.5rem;
+    font-weight: bolder;
+}
+
+.primary-outline,
+.primary {
+    color: #29438b;
```
Comment (tushartiwari7) [inline Components/Alert/alert.css:14]: store color variables on root and use them.

**Review by tushartiwari7 — APPROVED** [review]
Added some suggestions. Rest all looks fine!

File: Components/Alert/alert.css (hunk 2 above)
Comment (Manju-sree) [reply Components/Alert/alert.css:14]: i'm using different colors , so not maintained in root

**Pull request merged**